### PR TITLE
update vuefire example for 2.0

### PR DESCRIPTION
**Pull request by chrisvfritz**

### CODE REVIEW

**Comment by chrisvfritz** [comment]
@posva Would you be able to rebuild when you get a chance? It looks like an intermittent failure.

**Comment by posva** [comment]
@chrisvfritz Rebuilding does nothing 😭 It has been failing for a while. I think it's the firebase demo ref failing. I'll check further on Thursday. Maybe I need to use a real ref, maybe mocks exists,  I can build mine's too

File: examples/todo-app/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    Then you can open this file in your browser.
+    If you just prefer to see this example with
+    the latest published version of VueFire, you
+    play with the code in this fiddle:
+
+    https://jsfiddle.net/chrisvfritz/acy5n6j6/
+    -->
+
+    <div id="app">
+      <input
```
Comment (posva) [inline examples/todo-app/index.html]: Don't you like the form? 😄

**Comment by posva** [comment]
Thanks for updating the dependencies. It's funny, I was planning on doing some maintenance this we

**Comment by jwngr** [comment]
@posva - Yeah, we (Firebase) recently turned down the `firebaseio-demo.com` URLs, so you'll have to use a real Firebase project. I have one called `https://oss-test.firebaseio.com` which I used for all of our open source repos which you can feel free to piggy-back off of. Otherwise, you can just create any old project (e.g. `https://vuefire-tests.firebaseio.com`), set it to publicly readable and writable and use that just like you used demo. Sorry for the troubles!

File: examples/todo-app/index.html (hunk 1 above)
Comment (chrisvfritz) [inline examples/todo-app/index.html]: Ha, I was just trying to simplify a little. And now it's also more like the todo app example in the guide.

**Comment by posva** [comment]
@jwngr Ok, I was expecting that haha. I'll change the ref this  Thursday then. Thank you very much for tuning in 😄

**Pull request merged**